### ISSUE TITLE
keenslider within component renders only one slide

### ISSUE THREAD

**Issue by Memphis335** [rcbyr/keen-slider#8]
Hey!

I am trying to implement the slider `("keen-slider": "^4.2.1")` inside a component. This component then renders within a page component that has a `<section></section>`. The slide that I am trying to render is material cards with a min-width of 365px;

I want three to show in the center of the page max-width 80% on large screens and two on tablets and only one on mobiles. 

Similar to this: [codesandbox](https://codesandbox.io/s/keen-slider-breakpoint-angular-7yiou?).

The component renders only 1 card for some reason.

**Comment by rcbyr** [comment]
Hey,

I forked your codesandbox, update the version to 4.2.1, and fixed the error that happens on ngAfterViewInit() by wrapping the slider initialization with a setTimeout. It seems to work as you wanted to be. Can you confirm that?

[Codesandbox Fork](https://codesandbox.io/s/keen-slider-breakpoint-angular-sc796?file=/src/app/app.component.ts)

**Comment by rcbyr** [comment]
I just realised there is a bug for the inital height calculation. In some cases, I only see the image and not the text. I try to fix this tomorrow.

**Comment by Memphis335** [comment]
Hi @rcbyr,

Thank you for your assistance so far but the fixes did not help. I dug a little deeper and found the following : 

![image](https://user-images.githubusercontent.com/11742589/85108018-3fa12380-b20f-11ea-8fee-21480403bced.png)

I could not see it due to overflow: hidden, which is expected. The problem seem to be here `transform: translate3d(412px, 0px, 0px);`. 

So it appears that is showing all three just not on the same line.

**Comment by rcbyr** [comment]
@Memphis335 
Ah, now I understand. You didn't import the CSS file. Please, see the [installation instructions](https://keen-slider.io/#installation). You can also copy the content of the CSS file to the place where you want it to be.

**Comment by Memphis335** [comment]
I imported your stylesheet in scss like this (uncommented obviously)
![image](https://user-images.githubusercontent.com/11742589/85112287-b17c6b80-b215-11ea-85ae-9c0b85934746.png)
 which did seem to work. 

For completeness sake, I copied your styles to my file directly to no avail: 
![image](https://user-images.githubusercontent.com/11742589/85112333-c48f3b80-b215-11ea-8790-51a65d13e65c.png), 

I also saw in your options that I can pass the class of my slides but that doesn't seem to be doing anything. 
![image](https://user-images.githubusercontent.com/11742589/85112509-00c29c00-b216-11ea-8f1d-80186f51cf24.png)

I have renamed my classes to yours in the stylesheet `.keen-slider__slide` but now nothing shows.
